worlds/hk/Options.py

```diff
@@ -402,22 +402,30 @@ class WhitePalace(Choice):
     default = 0
 
 
-class DeathLink(Choice):
-    """
-    When you die, everyone dies. Of course the reverse is true too.
-    When enabled, choose how incoming deathlinks are handled:
-    vanilla: DeathLink kills you and is just like any other death.  RIP your previous shade and geo.
-    shadeless: DeathLink kills you, but no shade spawns and no geo is lost.  Your previous shade, if any, is untouched.
-    shade: DeathLink functions like a normal death if you do not already have a shade, shadeless otherwise.
+class ExtraPlatforms(DefaultOnToggle):
+    """Places additional platforms to make traveling throughout Hallownest more convenient."""
+
+
+class DeathLinkShade(Choice):
+    """Sets whether to create a shade when you are killed by a DeathLink and how to handle your existing shade, if any.
+
+    vanilla: DeathLink deaths function like any other death and overrides your existing shade (including geo), if any.
+    shadeless: DeathLink deaths do not spawn shades. Your existing shade (including geo), if any, is untouched.
+    shade: DeathLink deaths spawn a shade if you do not have an existing shade. Otherwise, acts like shadeless.
+
+    * This option has no effect if DeathLink is disabled.
     """
-    option_off = 0
-    alias_no = 0
-    alias_true = 1
-    alias_on = 1
-    alias_yes = 1
+    option_vanilla = 0
     option_shadeless = 1
-    option_vanilla = 2
-    option_shade = 3
+    option_shade = 2
+    default = 2
+
+
+class DeathLinkBreaksFragileCharms(Toggle):
+    """Sets if fragile charms break when you are killed by a DeathLink. All other deaths still break fragile charms.
```

Comment:
As noted by folks on discord it's probably slightly more accurate to say something like "fragile charms will still be broken by your own deaths as usual" to cover for cases where your own deaths actually will not break the charm